### PR TITLE
implemented API for configuring output value

### PR DESCRIPTION
**Pull request by unby**
Hello, it's atomaric changes for SetParameterValue method

### CODE REVIEW

**Comment by unby** [comment]
Hi, do you have any questions about the feature?

**Comment by dvoituron** [comment]
I don't yet time... But i'll check in some days.

File: DbMocker/Data/MockDbParameter.cs [hunk 1]
```diff
@@ -10,7 +10,7 @@ public class MockDbParameter : DbParameter
         private bool _hasDbTypeIsDefined = false;       // True when the DbType property is already setted and defined.
         private object _value = null;
         private DbType _dbType = DbType.AnsiString;
-
+        private DbParameter _parameter;
```
Comment (dvoituron) [inline DbMocker/Data/MockDbParameter.cs:13]: Why do you create this `DbParameter` if you don't use it later in the code?

**Review by unby — COMMENTED** [review]
I think it better to give up  MockDbParameter and use the original DbParameter. but it requires a big refactoring ...

**Review by dvoituron — APPROVED** [review]

**Comment by dvoituron** [comment]
Thanks... I've merged you branch to the master branch.
I'll create a new nuget package (v1.13) in some days, including your improvements.

**Comment by dvoituron** [comment]
Published in release 1.13

**Pull request merged**